### DISCUSSION TITLE
Handling change in device orientation?

### DISCUSSION THREAD

**Discussion by deeplook** [flet-dev/flet#2420] [Q&A] [answered]
### Question

I'm just getting started with Flet, but I wonder if and how I can get access to information about the local device like screen size and orientation or more in order to adjust the page layout? Specifically for orientation I see this in the Flutter docs, but I'm lost about how much of this is available in Flet, too?

https://docs.flutter.dev/cookbook/design/orientation

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by MaxXilon** [comment, marked as answer]
I believe the contents of this link is what you're looking for [documentation on the Page control in flet](https://flet.dev/docs/controls/page) .
The Page control in flet has these properties that can assist.
• width
• height
• on_resize, etc
Try researching on the Page control for more information.

**Reply by FeodorFitsner** [reply]
...and to decide whether it's portrait or landscape just compare width and height.
